pages/common/composer.md

```diff
@@ -0,0 +1,23 @@
+# composer
+
+> A per-project package manager for PHP.
+
+- Update composer to the latest version:
+
+`composer self-update`
+
+- Make a package a dependency for this project:
```

Comment:
I suppose we could use "the current project" instead, but that's still a bit unclear. What does the npm page do?